index.html

```diff
@@ -10932,6 +10951,63 @@ <h3>Graph Containers</h3>
     ignored for such documents.</p>
 </section>
 
+<section class="normative changed"><h3>Alternate Document Location</h3>
+
+  <p>Documents which can't be directly interpreted as JSON-LD can provide an alternate location containing JSON-LD.
+    One way to provide this is by using referencing a JSON-LD document in an <a>HTTP Link Header</a>.
```

Comment:
```suggestion
    One way to provide this is by referencing a JSON-LD document in an <a>HTTP Link Header</a>.
```

Reply:
Fixed in f1f4efe0c995b9ab901ed19cffdf7d286bd3961b.